packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -471,17 +471,24 @@ export default [
       exportRetentionExists: state =>
         some(
           state.newSchedules,
-          ({ exportRetention }) => +exportRetention !== 0
+          ({ exportRetention }) =>
+            exportRetention !== '' && +exportRetention !== 0
```

Comment:
Only store numeric values in the state, create a dedicate `NumberInput` component if necessary.